pygmt/tests/test_geopandas.py

```diff
@@ -64,3 +64,40 @@ def test_geopandas_info_shapely(gdf, geomtype, desired):
     geom = gdf.loc[geomtype].geometry
     output = info(table=geom, per_column=True)
     npt.assert_allclose(actual=output, desired=desired)
+
+
+@pytest.mark.mpl_image_compare
+def test_geopandas_plot_default_square():
+    """
+    Check the default behavior of plotting geopandas DataFrame with Point
+    geometry in 2d.
+    """
+    point = shapely.geometry.Point(5, 5)
+    gdf = gpd.GeoDataFrame(dict(geometry=[point]))
+    fig = Figure()
+    fig.plot(data=gdf, region=[0, 10, 0, 10], projection="X10", frame=True)
```

Comment:
To speed up the test, we try to keep the figure small-ish. There's a lot of whitespace with a 10x10 plot, so maybe trim it to 2x2cm instead.

```suggestion
    point = shapely.geometry.Point(1, 2)
    gdf = gpd.GeoDataFrame(geometry=[point])
    fig = Figure()
    fig.plot(data=gdf, region=[0, 2, 1, 3], projection="X2c", frame=True)
```